src/Symfony/Bundle/FrameworkBundle/CHANGELOG.md

```diff
@@ -32,7 +32,8 @@ CHANGELOG
  * Added a `InMemoryTransport` to Messenger. Use it with a DSN starting with `in-memory://`.
  * Added `framework.property_access.throw_exception_on_invalid_property_path` config option.
  * Added `cache:pool:list` command to list all available cache pools.
-
+ * Deprecated `KernelBrowser::getContainer` should not return null when the Kernel is not booted.
```

Comment:
I'm not sure I understand what this means - I'd suggest using the same sentence as in the deprecation message